lib/pages/setting/setting.dart

```diff
@@ -104,12 +104,12 @@ class _SettingPageState extends State<SettingPage> {
 
 class SettingList extends StatelessWidget {
   final List<SettingListItem> lists;
-  final String title;
+  final String? title;
```

Comment:
```suggestion
  final String title;
```

default 주니까 옵셔널 빼도 될듯~